extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -157,14 +159,69 @@ export class AmpStoryAudioSticker extends AMP.BaseElement {
       : FALLBACK_DEFAULT_STICKER;
   }
 
+  /**
+   * @private
+   */
+  maybeInitializeStickerStyle_() {
+    this.toggleStickerState_(
+      'outline',
+      this.element.getAttribute('sticker-style') === 'outline'
+    );
+    this.toggleStickerState_(
+      'dropshadow',
+      this.element.getAttribute('sticker-style') === 'dropshadow'
+    );
+
+    const outlineColor = this.element.getAttribute('outline-color');
+    if (this.isValidRGBColor_(outlineColor)) {
+      setImportantStyles(this.element, {
+        '--story-audio-sticker-outline-color': outlineColor,
+      });
+    }
+    const dropshadowColor = this.element.getAttribute('dropshadow-color');
+    if (this.isValidRGBColor_(dropshadowColor)) {
+      setImportantStyles(this.element, {
+        '--story-audio-sticker-dropshadow-color': dropshadowColor,
+      });
+    }
+  }
+
+  /**
+   * @private
+   * @param {string} RGBColor
+   * @return {boolean}
+   */
+  isValidRGBColor_(RGBColor) {
+    return RGBColor && RGBColor.match(/rgba?\((\d{1,3}), (\d{1,3}), (\d{1,3})/);
+  }
+
   /** @private */
   initializeListeners_() {
     this.element.addEventListener(
       'click',
       () => this.storeService_.dispatch(Action.TOGGLE_MUTED, false),
       true
     );
-    // TODO: add listeners for click animations.
+    this.element.addEventListener(
+      'pointerdown',
+      () => this.toggleStickerState_('clicking', true),
+      {passive: true}
+    );
+    this.element.addEventListener(
+      'pointerup',
```

Comment:
Try with the selector `button:active` instead. https://developer.mozilla.org/en-US/docs/Web/CSS/:active